programs/test/udp_proxy_wrapper.sh

```diff
@@ -108,7 +108,7 @@ if [ $VERBOSE -gt 0 ]; then
     echo "[ $SRV_BIN $* ]"
 fi
 
-"$SRV_BIN" "$@" >&2 &
+exec "$SRV_BIN" "$@" >&2 &
```

Comment:
I think you want to remove the following lines then, it doesn't make sense to keep unreachable statements around.